include/octo-logger-cpp/compat.hpp

```diff
@@ -0,0 +1,25 @@
+
+#ifndef COMPAT_HPP
+#define COMPAT_HPP
+#include <ctime>
+#ifdef _WIN32
+#include <cerrno>
+#endif
+
+namespace octo::logger::compat
+{
+
+inline struct tm* localtime(const time_t* timep, struct tm* result)
```

Comment:
why not use std chrono?

Reply:
there is no real replacement to std::localtime on c++ on the version we are on, using std::chrono will not help